exercises/grade-school/GradeSchoolTest.cs

```diff
@@ -1,61 +1,135 @@
+using GradeSchool;
```

Comment:
Yes, but then you didn't remove the namespace from your implementation. Just look at the default implementation: https://github.com/exercism/xcsharp/blob/master/exercises/grade-school/GradeSchool.cs